### DISCUSSION TITLE
Udevmount is executed after transmission and minidlna processes

### DISCUSSION THREAD

**Discussion by donacarr** [Freetz-NG/freetz-ng#368] [Allgemein] [answered]
I have two issues with udevmount even if I changed to Label the active mount method.
I am running with a configuration where I have minidlna and trasmission running.
The USB key has label "sandisk"

1)  it happens that a different  mount point is used (sandisk-1) when transmission is set to start automatically.
Below the options used:
```
_transmission-daemon --no-portmap --incomplete-dir /var/media/ftp/sandisk/bittorrent/download/incomplete --finish-dir /var/media/ftp/sandisk -P 54321 -L 120 -l 30_
```

2) minidlna does not scan USB key when starts (probably because the USB key has not been still mounted)
Here is the log
```
_[1970/01/01 01:01:27] scanner.c:731: warn: Scanning /var/media/ftp
[1970/01/01 01:01:32] scanner.c:820: warn: Scanning /var/media/ftp finished (0 files)!_
```
 
Of course the same fritz configuration worked fine with previous old freetz build (using Freetzmount)
I guess that both processes are started before Udevmount runs.
But need confirmation from some expert...
Any idea?
Thank you in advance

**Comment by fda77** [comment, marked as answer]
Avm changed some things, eg using this systemd-like thing and udevd, so boot/start order changed.  Udevmount/freetzmount provide the hooks for external depending on Fritzos.
This workaround could be used for all packages which need files on usb and/or a correct time:
- enable external in menuconfig, no need to select packages, but you could
- create a directory on the device, eg `mkdir /var/media/ftp/MYSTICK/external`
- create one or both of these files in this directory, im not sure anymore. i have both: 
- `touch /var/media/ftp/MYSTICK/external/.external`
- `touch /var/media/ftp/MYSTICK/external/external.pkg`
- if packages are selected in external-menuconfig, these files are created automatically in the external.tar
- configure as you like
![image](https://user-images.githubusercontent.com/471850/129607943-e4924de6-5456-4ee9-ac51-b6d9765215cf.png)
- after next boot you should see
![image](https://user-images.githubusercontent.com/471850/129608131-53edf339-456d-4c4f-83b8-281413cfe87e.png)
If you hit "unmount" on avm webif these packages are stopped before
Every dot in the "Waiting" lines are 1 second
